<launch>

  <!-- turtlesim node -->
  <node
      pkg = "turtlesim"
      type = "turtlesim_node"
      name = "turtlesim"
      respawn = "true"
      >
      <param name ="background_r" value="0" />
      <param name ="background_g" value="255" />
      <param name ="background_b" value="0" />
  </node>


  <!-- teleop Node -->
  <node
      pkg = "turtlesim"
      type = "turtle_teleop_key"
      name = "teleop_key"
      required = "true"
      launch-prefix = "xterm -e"
      />
  
</launch>




